examples/todos/src/main.rs

```diff
@@ -82,13 +82,11 @@ pub struct Pagination {
 }
 
 async fn todos_index(
-    pagination: Option<Query<Pagination>>,
+    Query(pagination): Query<Pagination>,
```

Comment:
This should not change I think.

Reply:
Ah, this was intentional but I guess it should really be a separate PR with its own description.